src/test/views/test_read_apriori.py

```diff
@@ -77,6 +102,31 @@ def test_returns_expected_pressure():
     ])
 
 
+@pytest.mark.parametrize("species,doy,latitude,source,expect", (
+    ('CO2', 1., -90., None, 0.0003626),
+    ('CO2', 1., 35., None, 0.0003626),
+    ('CO2', 15., 35., None, 0.0003626),
+    ('CO2', 180., 35., None, 0.0003626),
+    ('CO2', 350., 35., None, 0.0003626),
+    ('CO2', 365., 35., None, 0.0003626),
+    ('CO2', 365., 80., None, 0.0003626),
+    ('CO2', 365., 90., None, 0.0003626),
+    ('CO', 1., 80., 'mipas', 0.34654013),
+    ('CO', 10., 80., 'mipas', 0.3749516),
+    ('CO', 30., 80., 'mipas', 0.2280777),
+    ('CO', 335., 80., 'mipas', 0.1782076),
+    ('CO', 355., 80., 'mipas', 0.3118149),
+    ('CO', 365., 80., 'mipas', 0.3433833),
+))
+def test_get_apriori_co2_vmr_does_not_vary(
```

Comment:
The test includes both CO and CO2, but the VMR does vary for CO, but not CO2. Why is it supposed to vary for one, but not the other, and is the test appropriately named?

Reply:
Ah, then I see. I think a comment in the test would be good at least, so that when whoever looks at it next we can see the logic of it.

Reply:
(in which case the test name is fine)